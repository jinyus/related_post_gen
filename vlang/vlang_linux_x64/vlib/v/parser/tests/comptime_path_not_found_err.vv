fn main() {
	tmp := $tmpl('tempate.txt')
	a := tmp
}
