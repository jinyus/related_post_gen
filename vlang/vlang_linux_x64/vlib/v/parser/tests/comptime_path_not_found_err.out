vlib/v/parser/tests/comptime_path_not_found_err.vv:2:15: error: template file "tempate.txt" not found
    1 | fn main() {
    2 |     tmp := $tmpl('tempate.txt')
      |                  ~~~~~~~~~~~~~
    3 |     a := tmp
    4 | }
